avm/res/document-db/database-account/tests/e2e/plain/main.test.bicep

```diff
@@ -124,3 +122,181 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
     diagnosticDependencies
   ]
 }]
+
+module enableSystemAssignedManagedIdentity '../../../main.bicep' = {
```

Comment:
Please note - if you add multiple test deployments to this file, none of them will show up in the readme. Personally, I'd suggest to have a test file / folder each. I'd also be curious why not having the parameters you test all being part of the same test - unless they're not compatible. If the latter is the case, you could try to split seperate test cases off where required.

Reply:
@eriqua what are your thoughts on the changes? I guss ultimately it's up to the owner, but it would be nice to align with the convention, I guess.

Reply:
In my opinion, I don't mind if they don't appear in the Readme file. In fact, I am happy if they don't appear in it as I created these to have a more complete test suite rather than having more examples for the user. In the end, the lack of these tests is what provoked these bugs. 

Some of those settings are incompatible with each other so I can't test them all in only one account. Yes, I could create a test folder for each of them but think that I would then have a LOT of test folders since I have many tests for multiple scenarios and there are still many more which I didn't create just because the PR was already big.

Reply:
Another thing worth to mention is that is always easier to find bugs or tests features with tiny tests rather than big bang ones. Since it was very but very hard to find out what was happening with the analytical store + the time taken to deploy everything in one go with the pre PR sql test suite

Reply:
I will create all the tests folder then

Reply:
If you need a hand, just say the word

Reply:
Say no more :)

I added suggestions that would take care of the issue. Once applied, you should be able to run the script without an issue :)

For transparency, this would be the corresponding spec: https://azure.github.io/Azure-Verified-Modules/specs/bicep/#id-bcpnfr12---category-testing---deployment-test-naming

Reply:
> @eriqua what are your thoughts on the changes? I guss ultimately it's up to the owner, but it would be nice to align with the convention, I guess.

I see the discussion progressed to an agreement for dedicated test folders already. Agree with the agreement 🤝 😆